### PR TITLE
integration_testing_mavsdk: updated cli args

### PR DESCRIPTION
**Pull request by julianoes**
There is now a json config file that needs to be selected.

Depends on https://github.com/PX4/Firmware/pull/14296.

### CODE REVIEW

File: en/test_and_ci/integration_testing_mavsdk.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Download the matching .deb file (whichever is correct for your system) from [MAV
 Install it using `dpkg`, e.g.:
 
 ```sh
-sudo dpkg -i mavsdk_0.23.0_ubuntu18.04_amd64.deb
+sudo dpkg -i mavsdk_0.24.0_ubuntu18.04_amd64.deb
```
Comment (hamishwillee) [inline en/test_and_ci/integration_testing_mavsdk.md]: @julianoes I've removed all of these installation instructions and lined to the mavsdk ones in https://github.com/PX4/Devguide/pull/975
Can you merge that and update this appropriately.

If you disagree with the strategy we can discuss further

File: en/test_and_ci/integration_testing_mavsdk.md [hunk 2]
```diff
@@ -77,10 +77,10 @@ DONT_RUN=1 make px4_sitl gazebo mavsdk_tests
 
 ### Run All PX4 Tests
 
-To run all tests, use:
+To run all SITL tests as defined in [sitl.json](https://github.com/PX4/Firmware/blob/master/test/mavsdk_tests/configs/sitl.json), do:
 
 ```sh
-test/mavsdk_tests/mavsdk_test_runner.py --speed-factor 20 --gui
+test/mavsdk_tests/mavsdk_test_runner.py test/mavsdk_tests/configs/sitl.json --speed-factor 10
```
Comment (hamishwillee) [inline en/test_and_ci/integration_testing_mavsdk.md:29]: This looks great. Is `mavsdk_test_runner.py` documented anywhere? Ie. arguments? Would be good to add a note or similar explaining either what the options are, or noting that you can get them using (say) `mavsdk_test_runner.py -h`

**Review by hamishwillee — COMMENTED** [review]
See comments, in particular PR for linking to mavsdk installation docs rather than having in place.

Note, if you have to keep the instructions here, I'd work out a strategy that means you don't have to keep updating the version numbers.

File: en/test_and_ci/integration_testing_mavsdk.md (hunk 2 above)
Comment (julianoes) [reply en/test_and_ci/integration_testing_mavsdk.md:29]: See just below. The arguments will change over time, I wouldn't document them in detail but just mention `-h`.

**Comment by hamishwillee** [comment]
Yes looks great.

**Pull request merged**